build/package/Dockerfile

```diff
@@ -19,7 +19,11 @@ ENV GOOS=linux \
 WORKDIR /gotenberg
 
 # Copy our source code.
-COPY . .
+COPY internal ./internal
+COPY cmd ./cmd
+COPY go.sum go.sum
+COPY go.mod go.mod
+
```

Comment:
As this is an "intermediate" image, is this really necessary? Or it's just for speeding up the build process?

Reply:
No, it's not *really* necessary - just a result of our own habit of never copying more than is absolutely necessary.

Reply:
No, it's not *really* necessary - just a result of our own habit of never copying more than is absolutely necessary to facilitate best use of the Docker cache.